go.mod

```diff
@@ -17,19 +18,17 @@ require (
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
-	github.com/tendermint/tendermint v0.34.12
+	github.com/tendermint/tendermint v0.34.13
 	github.com/tendermint/tm-db v0.6.4
 	google.golang.org/genproto v0.0.0-20210114201628-6edceaf6022f
-	google.golang.org/grpc v1.37.0
-	google.golang.org/protobuf v1.26.0
+	google.golang.org/grpc v1.40.0
+	google.golang.org/protobuf v1.27.0
 	gopkg.in/yaml.v2 v2.4.0
 )
 
 replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
 
-replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.2-alpha.regen.4
-
-replace github.com/tendermint/tendermint => github.com/tendermint/tendermint v0.34.12
```

Comment:
Why was this replace removed?

Reply:
you don't need to replace tendermint - it's already fixed in the "require" section.

Reply:
We should still be doing this just in case dependencies use other versions, no?

Reply:
Yes - it replaces all versions. It looks like a hack. You are depending on your own version of SDK - maybe it's better to fix the version you want there? Anyway - I will rollback that change and in the future you can consider removing it.